src/screens/UserPortal/Home/Home.tsx

```diff
@@ -90,8 +90,8 @@ export default function home(): JSX.Element {
   const [filteredAd, setFilteredAd] = useState<InterfaceAdContent[]>([]);
   const [showModal, setShowModal] = useState<boolean>(false);
   const { orgId } = useParams();
-  const organizationId = orgId?.split('=')[1] || null;
-  if (!organizationId) {
+  console.log(typeof orgId, orgId);
```

Comment:
Can avoid logging the data (Tests get very long console due to this)

Reply:
Yess forgot about it